.gitignore

```diff
@@ -1,2 +1,5 @@
 **/__pycache__
 **/.ipynb_checkpoints
+**/*.log
+**/*.ipynb
```

Comment:
I'm not sure we want to exclude all notebooks do we? I can imagine we may eventually want to use them for viz or blog post-type purposes. If you have some in your local directory you'd like to ignore, you might want to consider using the `.git/info/exclude` file as described [here](https://stackoverflow.com/questions/1753070/how-do-i-configure-git-to-ignore-some-files-locally)

Reply:
Yeah this was a local problem. Thanks for the pointer